src/View/Helper/FormHelper.php

```diff
@@ -1649,7 +1649,7 @@ public function postLink($title, $url = null, array $options = [])
 
         $formName = str_replace('.', '', uniqid('post_', true));
         $formOptions = [
-            'action' => $this->Url->build($url),
+            'action' => Router::url($url),
```

Comment:
So, it not a problem here, but in the SecurityComponent, since the request is being blackholed?